packages/salesforcedx-vscode-lightning/package.nls.json

```diff
@@ -1,7 +1,9 @@
 {
   "lightning_open_component_title": "SFDX: Open Lightning Component",
   "lightning_explorer_title": "Lightning Explorer",
-  "feature_previews_title": "Salesforce Feature Previews",
+  "feature_previews_title": "Salesforce Lightning ",
```

Comment:
Not sure what we should be naming our preferences in vscode...I was using this previews setting, but I think we want to have our own section now that there will be more than one preference. Should have have preferences in a single section for both LSPs or one common section?